R/oc_forward.R

```diff
@@ -340,15 +340,29 @@ oc_forward <-
 #' oc_forward_df(df2, placename = locations,
 #'               countrycode = countrycode)
 #' }
+oc_forward_df <- function(...) UseMethod("oc_forward_df")
 
-oc_forward_df <-
+# No @name so it does not show up in the docs.
```

Comment:
Why not `@NoRd`?

Reply:
Dunno. Not specifiying `@name` is sufficient for it to not show up in docs, so `@NoRd` is redudant in a way, but I agree that it conveys intent more succinctly.

Reply:
as you prefer!

Reply:
Already made the change.

Reply:
(and I thought, I had already merged, but apparently not)